finance.md

```diff
@@ -0,0 +1,69 @@
+
+Hypha Worker Co-operative is a worker co-operative incorporated in Ontario, Canada. The business has seven Founding Members, four of which are First Directors with signing authority. As a non-share-capital worker co-operative, we are a non-profit organization, and every Member holds both a Worker and an Owner relationship with the business.
+
+## Financial accounts
+
+Our main financial institution is **Desjardins** and our main funds are stored in a business chequing account, where all outgoing transactions require 2 signatures from Members of the Finance WG.
+
+Hypha is expected to conduct business with clients from other countries and is ready to [accept funds in CAD, USD, and EUR](https://transferwise.com/help-smart/11/getting-started/2571907/what-currencies-do-you-support). We use **TransferWise** to minimize transaction and currency exchange fees. Our Transferwise account has local accounts in Canada, United States, and Europe, so we can bill clients as a local financial institution in their local currency. Our TransferWise account does not require multiple signatures, and the Finance WG is responsible for transfering money into our main Desjardins account when accumulated funds exceed a CAD 10,000 threshold.
+
+## Invoicing a client
+
+Hypha charges clients on a sliding scale, and projects may be billed hourly, weekly, monthly, or per project, depending on the specifics of each project. For all projects, regardless of how a customer is charged, Members log their hours using **Clockify**. For clients and billing models that require timesheets, Clockify is used to generate these timesheets to calculate the amount to invoice each period.
+
+### Generating timesheets
+
+1. Generate a per-member timesheet in Clockify and save (e.g. [aether_201908_bl](https://clockify.me/bookmarks/5dab94738a17084c0337690d) and [aether_201908_yj](https://clockify.me/bookmarks/5dab94828a17084c03376911))
+1. Generate a team timesheet in Clockify and save (e.g. [aether_201908](https://clockify.me/bookmarks/5dab94a0ae125662897c69b4))
+1. Export team timesheet as PDF to be sent to client, if applicable
+
+This process is done using the Clockify [Summary Report](https://clockify.me/reports/summary). Saved reports are found under [Saved Reports](https://clockify.me/reports/saved).
+
+### Generating invoices
+
+1. Generate an invoice in **Quickbooks Online** based on the pricing structure (e.g. timesheet hours and project rate)
+1. Add operational cost line item, if applicable
+1. Add GST, if applicable
+
+The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
+
+Invoices (and timesheets, if requested) are sent to clients and are payable in CAD, USD, or EUR. Invoices are due 14 days after the invoice date, after which a 5% late fee will be charged for each additional week.
```

Comment:
I've voiced this a few times, but I've been told in a few places that it's better practice to inflate (service fee or whatever), and offer a discount for early payment. My feeling is that there's zero cost to doing it this way, and the framing is inherently friendlier when the inevitable late-payments happen -- we are less likely to hear "please, I'm only 10 hours late, don't ding me", because it's framed as a favour we're offering them, not a penalty.

Reply:
I have read your **discount for early payment** proposal before and you did mention it a couple time earlier, but I feel it doesn't fit well into the way we bill at the moment. So far our work has been on a negotiated hourly rate / fixed project budget, it seems weird to on the bill inflate that amount and then discount it.

Also agree with @YurkoWasHere's points about not actually exercising the late fee, and having cumulative over time.

@patcon if there are particular projects where you quoting / invoicing with the discount structure is suitable, I'd just do that on a project basis, but I don't feel it's a suitable org-wide strategy.

Reply:
I agree overarchingly with Yurko's points about connection b/w discount and late payment. I'm used to inflation/discounts being treated as optics separate from late payment (which I think we should have flexible policies on until we develop a sense of what works for us).

My general take is to be upfront/fixed with our costs (relying on community pricing models) over a payment discounts. Previous company I worked at with many small contracts b2b never had a practice discounting, I can say from experience that any changes to pricing structure were met with confusion and frustration by usually the limited set of clients who always took more touch to deal with.

All that said. I think the proposed late policy from @benhylau has pretty tight term+high percentage that I would soften (I'm not used to things kicking in until _overdue_ by a set time after a peroid). I think also setting up a payment schedule in contracts is probably the best way to handle getting paid over fees as a mechanism

Reply:
Could we agree to just make this a guideline?

If I'm running a small client through the co-op, I just want to be able to choose my own language and framing that suits my sense of human-centred relationships. I like a gift more than a penalty. I don't mind if others don't want to do late fees on their projects or larger ones, I just would prefer this not pushed on my own initiatives.

Is that possible?

Reply:
Speaking of which, thoughts on using IEEE spec language throughout our policies? SHALL MUST SHOULD etc

Reply:
```suggestion
Invoices (and timesheets, if requested) SHOULD be sent to clients and SHALL be payable in CAD, USD, or EUR. Invoices SHOULD be due 14 days after the invoice date, after which a 5% late fee SHOULD be charged for each additional week.
```

Reply:
> I think we should have flexible policies on until we develop a sense of what works for us

:+1: or also, i'd be fine to use IETF lingo as that flexibility.

> I can say from experience that any changes to pricing structure were met with confusion and frustration

:+1: agreed. imho changes to running contracts SHOULD be avoided, and we SHOULD try to preserve agreements a client started with. Would rather be able to handle variations than assume we'll decide/maintain globally consistent terms

> pretty tight term+high percentage that I would soften

:+1: but no block. pls someone suggest better numbers from experience. otherwise, happy to leave ben's. if it's too high, we'll hear about it and can adapt.

> I'm not used to things kicking in until overdue by a set time after a peroid

:-1: In favour of ben's single date over the multiple implied here: "pay by" date and then "overdue" date. KISS

> setting up a payment schedule in contracts is probably the best way to handle getting paid over fees as a mechanism

can you offer a clear example of the distinction?

Reply:
I have left the late fee % non-specific and changed to "_may_ be charged" to address concerns from @dcwalk and @hyphacoop/infra-ops-wg 

I also don't want two dates of payment. After "pay by" it is immediately "overdue".

re: IETF lingo, I am open to but feel it should be adopted across the broadly handbook if we decide to do so, so let's leave for future PRs.

>@patcon if there are particular projects where you quoting / invoicing with the discount structure is suitable, I'd just do that on a project basis, but I don't feel it's a suitable org-wide strategy.

I want to warn against diverging practices in general, as it could create more administrative burden we haven't considered, but we should discuss this when it comes time to make a proposal.

Reply:
Just wanted to note that I'm not +1 on the attempt to "avoid divergent practices". I feel quite differently -- that we should usually protect space for divergent practices to exist for a time, and only converge later.

But +1 to the rest! Thanks!